cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,52 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('/automation-practice-form');
   });
 
-  it('', () => {
-
+  it('Student Registration Form - autofilling', () => {
+    // asercja poprawności miejsca
+    cy.get('.main-header').contains('Practice Form').should('be.visible');
+    // wypełnienie formularza
+    cy.get('[placeholder="First Name"]')
```

Comment:
Most of the data should be randomized every time a test is run. You can achieve that by adding a `generateUser()` custom function to support folder or config file. It is suggested to use faker.